metaworld/policies/mujoco/sawyer_xyz/sawyer_door.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+from metaworld.policies.base import PID
+from metaworld.policies.mujoco.sawyer_xyz.base import SawyerXYZPolicy
+
+
+class SawyerDoorOpenV1Policy(SawyerXYZPolicy):
+    def __init__(self):
+        super().__init__(PID(10., 0., 0., stateful=False))
+
+    def xyz_goal(self, obs):
+        """
+        obs[:3]     The current xyz coord of end effector
```

Comment:
If you are going to document your methods, please use Google-style docstrings as detailed in CONTRIBUTING.md